hack/Dockerfile.kubectl

```diff
@@ -0,0 +1,43 @@
+# Copyright YEAR The KubeStellar Authors.

```

Comment:
Sounds like any `YEAR`?

Reply:
@waltforme thank you for catching it. Fixed in the commit.